### ISSUE TITLE
July release - Persistence of personalization limits

### ISSUE THREAD

**Issue by jasf-** [NEU-Libraries/scholar-onesearch#196]
Just a heads-up for the testing crew - please take a careful look at the personalization limits.  Buggy?

Personalization limits are displayed in these two places:

![stickinessofperslimits](https://cloud.githubusercontent.com/assets/4661920/8698144/27c6f082-2ac9-11e5-8e59-445050cfc6b9.png)

I can 'x' out the 'personalization' shown next to the relevance dropdown, but the personalization limit shown on top left column is still there.  Does x-ing out the personalization remove the limit?  I would think it does, and it appears to based on how ranking of search results changes - but the personalization limit shown on top left column is still there.  Confusing!

Also, sometimes I apply a personalization limit in the left column.  The limit in the left column then displays, but the 'personalization' shown next to the relevance dropdown does not.  

![dondeesta](https://cloud.githubusercontent.com/assets/4661920/8698129/efbf20ba-2ac8-11e5-9f3e-20df81d8ca8d.png)

If I run a new search, the 'personalization' next to relevance dropdown does appear.  So maybe the personalization limits do NOT take effect until a search is run or re-run.  This is also confusing; I would expect limits to work like the facets, which do not require a fresh search to take effect.

TL;DR - Seems buggy to me, and quite likely something to report to Ex Libris, but I'm still trying to work out what I'm seeing.  Additional eyeballs appreciated.
